doc-examples/hello-world-groovy/gradle/license.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'com.github.hierynomus.license'
+
+license {
+    header = rootProject.file('config/HEADER')
+    strictCheck = true
+    ignoreFailures = true
+    mapping {
+        kt     = 'SLASHSTAR_STYLE'
+        java   = 'SLASHSTAR_STYLE'
```

Comment:
Examples don't need license, and in that case they could use the one in the root directory.

Reply:
Yes, I agree. One of the micronaut plugins created these files. I will deal with this and remove the generation of these files.